client/src/components/Footer.js

```diff
@@ -15,28 +15,42 @@ const Footer = styled.footer`
 `;
 
 const List = styled.ul`
+  width: 100%;
   display: flex;
-  felx-direction: row;
+  flex-direction: row;
   justify-content: space-between;
 
   @media (max-width: 900px) {
     flex-direction: column;
+    justify-content: space-evenly; /* I tried to use this to create some space when
+    screen size is small but it didn't work..*/
   };
 `;
 
 const ListItem = styled.li`
+  width: 20%;
   display: flex;
-  felx-direction: row;
+  flex-direction: row;
+  justify-content: space-around;
   color: ${props => props.theme.textColor1};
-  
-  @media (min-width: 768px) {
+  @media only screen and (max-width: 995px) {
+    width: 100%;
+    flex-direction: column;
+    justify-content: space-between;
```

Comment:
do we need this?

I would just give the h6 some margin-right

now regarding margins and paddings, I call them spaces, so lets create in the theme variables like:

`space_s: 5px` 
`space_m: 10px` 
`space_l: 15px` 
`space_xl: 20px` 

we will play with the exact values to see what works best

Reply:
one important point about variables. I remember you said that since we in JS land we can not write `space-s`, so you did camelCase `spaceS`, thing is, maybe its just me, bu I find camelCasing somewhat ugly to read, how about we do kebab_case for those variables?

also usually when creating constants in JS they are all UPPER_CASE like that, I dont mind about lower or upper really, you decide, as long as they are easy to read,

Reply:
When working on `Theme.js`, since it's a "vanilla" `.js` file that does not import `styled-components`, I couldn't use kebab-case to name the keys of the object.
But the lodash spacing works fine. I will modify it in `Theme.js` but in a new branch.